web/src/components/ProductListPage/ProductListPage.tsx

```diff
@@ -243,18 +243,36 @@ export const getProductListPage =
 						/>
 
 						{documents.length === 0 ? (
-							<p id="results">
-								We can&apos;t find any matching products. Try{" "}
-								<Link
-									to={
-										pathname +
-										(ps && Number(ps) != defaultPageSize ? `?ps=${ps}` : "")
-									}
-								>
-									clearing your filters
-								</Link>{" "}
-								and starting again.
-							</p>
+							<div id="results">
+								<h2>No results found</h2>
+								<p>
+									Sorry, we haven’t found any results for you. You could try:
+								</p>
+
+								<ul>
+									<li>
+										Using the <Link href="/search">full site search</Link>.
+									</li>
```

Comment:
Could we change this `Link' component to a normal html anchor/a tag? We've found the link component to be a little problematic when linking off to non-next.js pages.